### PR TITLE
Feature: New Wallet User Onboarding Flow

### PR DESCRIPTION
**Pull request by CypherPoet**

### CODE REVIEW

**Comment by Christewart** [comment]
It looks like there is a bug where the tor & blockchain information are not formatted correctly on the page

Screen Shot 2022-09-14 at 7 03 51 PM
Screen Shot 2022-09-14 at 7 03 58 PM

**Comment by CypherPoet** [comment]
@Christewart Should be fixed by https://github.com/bitcoin-s/bitcoin-s-ts/pull/207/commits/8d05df2f9666fe1faf026b419f541ff8fc121b10

File: wallet-server-ui/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (user411) [inline wallet-server-ui/.prettierrc:1]: Delete file

File: wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            <!-- Step 1 -->
+
+            <button
+              class="card-slide-container__card__action-button"
+              *ngIf="onboardingStep.stepNumber == 1"
+              mat-flat-button
+              color="primary"
+              (click)="onImportExport()"
+            >
+              Back Up Now
```
Comment (user411) [inline wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html:52]: Move to translation key

File: wallet-server-ui/src/app/component/wallet-balance/wallet-balance.component.html [hunk 3]
```diff
@@ -3,18 +3,35 @@
     <h2 translate>walletBalance.heading</h2>
```
Comment (user411) [inline wallet-server-ui/src/app/component/wallet-balance/wallet-balance.component.html:3]: Please remove all file changes if no functional changes.

File: wallet-server-ui/src/service/websocket.service.ts [hunk 4]
```diff
@@ -16,7 +16,6 @@ import { ConfirmationDialogComponent } from '~app/dialog/confirmation/confirmati
 import { AddressService } from './address.service'
```
Comment (user411) [inline wallet-server-ui/src/service/websocket.service.ts:16]: Please remove changes to this file.

File: wallet-server-ui/src/styles.scss [hunk 5]
```diff
@@ -312,7 +312,7 @@ mat-form-field {
 .execute-container {
```
Comment (user411) [inline wallet-server-ui/src/styles.scss:312]: Please remove changes to this file.

File: wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html (hunk 2 above)
Comment (CypherPoet) [reply wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html]: Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae

File: wallet-server-ui/src/app/component/wallet-balance/wallet-balance.component.html (hunk 3 above)
Comment (CypherPoet) [reply wallet-server-ui/src/app/component/wallet-balance/wallet-balance.component.html]: Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae

File: wallet-server-ui/.prettierrc (hunk 1 above)
Comment (CypherPoet) [reply wallet-server-ui/.prettierrc]: Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae

File: wallet-server-ui/src/service/websocket.service.ts (hunk 4 above)
Comment (CypherPoet) [reply wallet-server-ui/src/service/websocket.service.ts:16]: Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae

File: wallet-server-ui/src/styles.scss (hunk 5 above)
Comment (CypherPoet) [reply wallet-server-ui/src/styles.scss]: Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae

**Comment by Christewart** [comment]
Can we make the graphics on these pages smaller? I have to scroll to see the network information and the tor address. As a new user, i don't think i would even know these are on the page

Screen Shot 2022-09-16 at 8 12 13 AM
Screen Shot 2022-09-16 at 8 12 24 AM

File: wallet-server-ui/src/styles.scss [hunk 6]
```diff
@@ -411,4 +411,4 @@ mat-form-field {
 }
 .even {
   // no styling yet
-}
+}
```
Comment (user411) [inline wallet-server-ui/src/styles.scss:414]: Can this file come off of the PR?

File: wallet-server-ui/src/service/websocket.service.ts [hunk 7]
```diff
@@ -280,4 +280,4 @@ export class WebsocketService {
     }
   }
 
-}
+}
```
Comment (user411) [inline wallet-server-ui/src/service/websocket.service.ts:283]: Can this file come off of the PR?

File: wallet-server-ui/src/service/wallet-state-service.ts [hunk 8]
```diff
@@ -569,4 +569,4 @@ export class WalletStateService {
     }))
   }
 
-}
+}
```
Comment (user411) [inline wallet-server-ui/src/service/wallet-state-service.ts:572]: Can this file come off of the PR?

File: wallet-server-ui/.gitignore [hunk 9]
```diff
@@ -40,6 +40,7 @@ npm-debug.log
 yarn-error.log
```
Comment (user411) [inline wallet-server-ui/.gitignore:40]: Can this file come off of the PR?

File: .gitignore [hunk 10]
```diff
@@ -1,2 +1,8 @@
 node_modules/
 
+# IDE - VSCode
+.vscode/*
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+.history/*
```
Comment (user411) [inline .gitignore:8]: What has a .history?

File: wallet-server-ui/.gitignore (hunk 9 above)
Comment (CypherPoet) [reply wallet-server-ui/.gitignore:40]: I was thinking it might be useful to add the `.prettierrc` line

**Comment by user411** [comment]
There is always scroll area reserved regardless of how I resize the window.
Screen Shot 2022-09-16 at 10 22 18 AM

File: .gitignore (hunk 10 above)
Comment (CypherPoet) [reply .gitignore:8]: I think this was generated when I ran something using Angular CLI

(Related: https://stackoverflow.com/questions/63988454/is-it-safe-to-remove-the-history-folder-when-committing-to-git)

File: wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+                class="card-slide-container__card__info-widget"
+              >
+                <h2
+                  class="info-widget__heading"
+                  translate
+                >
+                  newUserOnboarding.card.step4.currentBlockHeight
+                </h2>
+
+                <div
```
Comment (user411) [inline wallet-server-ui/src/app/component/new-user-onboarding/new-user-onboarding.component.html:121]: *ngIf="walletStateService.info"

**Comment by CypherPoet** [comment]
> There is always scroll area reserved regardless of how I resize the window. Screen Shot 2022-09-16 at 10 22 18 AM

Addresses this with https://github.com/bitcoin-s/bitcoin-s-ts/pull/207/commits/6c7e327e14cf668145691b1407f5549f112832f7

**Comment by user411** [comment]
Got half of it for me
Screen Shot 2022-09-16 at 12 14 35 PM

**Pull request merged**